statsmodels/sandbox/formula.py

```diff
@@ -549,13 +549,13 @@ def __mul__(self, other, nested=False):
         selftermnames = self.termnames()
         othertermnames = other.termnames()
 
-        I = len(selftermnames)
+        n_term_names = len(selftermnames)
         J = len(othertermnames)
 
         terms = []
         termnames = []
 
-        for i in range(I):
+        for i in range(n_term_names):
```

Comment:
in general, our convention has been and is
`n_xxx` is for observations or similar   (rows)
`k_xxx` for params, regressors or similar   (columns)